jwt.js

```diff
@@ -139,23 +154,28 @@ function fastifyJwt (fastify, options, next) {
         secretCallbackSign(reply.request, payload, callback)
       },
       function sign (secretOrPrivateKey, callback) {
-        jwt.sign(payload, secretOrPrivateKey, signOptions, callback)
+        jwt.sign(payload, secretOrPrivateKey, options, callback)
       }
     ], next)
   }
 
-  function requestVerify (verifyOptions, next) {
-    if (typeof verifyOptions === 'function') {
-      next = verifyOptions
-      verifyOptions = {}
+  function requestVerify (options, next) {
+    const badRequestError = new BadRequest('Format is Authorization: Bearer [token]')
```

Comment:
This is not ok. We should create the error only when needed. Move _just_ the string to the top of the file.